---
title: KGBassembler
summary: Karyotype-based genome assembly for Brassicaceae species
tags:
- Algorithms
- Brassicaceae Genetics
- Chromosome Painting
- Computational Biology Methods
- Computer Graphics
- Contig Mapping
- Genome
- Plant
- Karyotyping
- Sequence Alignment
- User-Computer Interface
date: "2012-09-01T00:00:00Z"

# Optional external URL for project (replaces project detail page).
external_link: ""

#image:
#  caption: Photo by rawpixel on Unsplash
#  focal_point: Smart

links:
#- icon: twitter
#  icon_pack: fab
- name: PubMed
  url: https://www.ncbi.nlm.nih.gov/pubmed/23044544
url_code: ""
#url_pdf: ""
#url_slides: ""
#url_video: ""

# Slides (optional).
#   Associate this project with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
# slides: example
---

The Brassicaceae family includes the most important plant model Arabidopsis thaliana and many cruciferous vegetable crops. Due to the lack of genetic and/or physical maps for many non-model Brassicaceae species, de novo assembly of full chromosomes is difficult in many experiments designed with short sequencing data. To address this problem, we developed a bioinformatics tool named KGBassembler to automatically finalize assembly of full chromosomes from contigs and/or scaffolds based on available karyotypes of Brassicaceae species (Figure 1). KGBassembler has been successfully applied to the assembly of Eutrema salsugineum, an extremophile model for abiotic stress tolerance studies.

* [**View Details**](http://bioinfo.nwafu.edu.cn/publication/bioinformatics_2012_kgbassembler/)
* **Please cite the following article**:<br>

> Ma, C., Chen, H., Xin, M., Yang, R., Wang, X. (2012). KGBassembler: a karyotype-based genome assembler for Brassicaceae species Bioinformatics  28(23), 3141-3143. https://dx.doi.org/10.1093/bioinformatics/bts586
