---
title: deepTS
summary: Exploring transcriptional switches from pairwise, temporal, and population RNA-Seq data using deepTS
tags:
date: "2020-06-01T00:00:00Z"

# Optional external URL for project (replaces project detail page).
external_link: ""

#image:
#  caption: Photo by rawpixel on Unsplash
#  focal_point: Smart

links:
#- icon: twitter
#  icon_pack: fab
- name: PubMed
  url: https://www.ncbi.nlm.nih.gov/pubmed/
url_code: "https://github.com/cma2015/deepTS"
#url_pdf: ""
#url_slides: ""
#url_video: ""

# Slides (optional).
#   Associate this project with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
# slides: example
---

**deepTS** is a powerful and flexible web-based Galaxy platform for identifying, visualizing and analyzing transcriptional switch (TS) events from pairwise, temporal and population transcriptome data. deepTS consists of three main functional modules, covering the processes of read cleaning and mapping, transcriptome map construction, expression abundance estimation, multiple-condition TS analysis pipelines, multiple-level TS characterization, and multiple-form visualization to allow users to perform TS analysis using either raw RNA-Seq data or expression abundance matrix directly.

* [**Web Server**](http://deepts.nwafu.edu.cn)
* [**View Details**](http://bioinfo.nwafu.edu.cn/publication/brief_bioinform_2020_deepts/)
* **Please cite the following article**:<br>

> Qiu Z., Chen, S., Qi Y., Liu C., Zhai J., Xie S., Ma, C. (2019). Exploring transcriptional switches from pairwise, temporal, and population RNA-Seq data using deepTS Briefings in Bioinformatics https://dx.doi.org/
