---
title: RSGCC
summary: Gini-based transcriptiome analysis
tags:
- Algorithms
- Arabidopsis Genetics
- Computational Biology methods
- Databases
- Nucleic Acid
- Gene Expression Profiling Methods
- Gene Expression Regulation
- Plant
- GeneS
- RNA
- Plant Analysis
- Plant Genetics
- ROC Curve
- Reproducibility of Results
- Sensitivity and Specificity
- Sequence Analysis
- RNA Methods
- Statistics as Topic Methods
- Transcription Factors Genetics
- Transcriptome
date: "2012-09-16T00:00:00Z"

# Optional external URL for project (replaces project detail page).
external_link: ""

#image:
#  caption: Photo by rawpixel on Unsplash
#  focal_point: Smart

links:
#- icon: twitter
#  icon_pack: fab
- name: PubMed
  url: https://www.ncbi.nlm.nih.gov/pubmed/22797655
url_code: "https://github.com/cma2015/rsgcc"
#url_pdf: ""
#url_slides: ""
#url_video: ""

# Slides (optional).
#   Associate this project with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
# slides: example
---

We introduced the Gini correlation coefficient, a member of the family of Gini methodologies that have been widely used in economics, to infer non-linear transcriptional regulatory relationships in transcriptomics data (Figure 2). The Gini-based R package **rsgcc** would be an alternative option for biologists to perform clustering analyses of gene expression patterns or transcriptional network analysis.

* [**View Details**](http://bioinfo.nwafu.edu.cn/publication/plant_physiol_2012_rsgcc/)
* **Please cite the following article**:<br>

> Ma, C., Wang, X. (2012). Application of the Gini Correlation Coefficient to Infer Regulatory Relationships in Transcriptome Analysis Plant Physiology  160(1), 192-203. https://dx.doi.org/10.1104/pp.112.201962
