---
title: "Functional assessment of human enhancer
activities using whole-genome STARR-sequencing"
authors:
- Yuwen Liu
-  Shan Yu
- Vineet K. Dhiman
- Tonya Brunetti
- Heather Eckart
- Kevin P. White
date: "2017T00:00:00Z"
doi: " 10.1186/s13059-017-1345-5"

# Schedule page publish date (NOT publication's date).
publishDate: "2017T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: "*Genome Biology*"
publication_short: "Genome Biology"

abstract: "**Background**: Genome-wide quantification of enhancer activity in the human genome has proven to be a challenging problem. Recent efforts have led to the development of powerful tools for enhancer quantification. However, because of genome size and complexity, these tools have yet to be applied to the whole human genome.
**Results**: In the current study, we use a human prostate cancer cell line, LNCaP as a model to perform whole human
genome STARR-seq (WHG-STARR-seq) to reliably obtain an assessment of enhancer activity. This approach builds upon
previously developed STARR-seq in the fly genome and CapSTARR-seq techniques in targeted human genomic
regions. With an improved library preparation strategy, our approach greatly increases the library complexity per unit of
starting material, which makes it feasible and cost-effective to explore the landscape of regulatory activity in the much
larger human genome. In addition to our ability to identify active, accessible enhancers located in open chromatin
regions, we can also detect sequences with the potential for enhancer activity that are located in inaccessible, closed
chromatin regions. When treated with the histone deacetylase inhibitor, Trichostatin A, genes nearby this latter class of
enhancers are up-regulated, demonstrating the potential for endogenous functionality of these regulatory elements.
**Conclusion**: WHG-STARR-seq provides an improved approach to current pipelines for analysis of high complexity
genomes to gain a better understanding of the intricacies of transcriptional regulation."

# Summary. An optional shortened abstract.
#summary: 

tags:
#- Epigenesis
#- Genetic
#- High-Throughput Nucleotide Sequencing
#- Plants Genetics
#- RNA
#- RNA Processing
#- Post-Transcriptional
#- Software
#- Transcriptome
featured: false

links:
- name: PubMed
  url: https://pubmed.ncbi.nlm.nih.gov/29151363/
#url_pdf: ''
#url_code: 'https://github.com/cma2015/PEA'
#url_dataset: ''
#url_poster: ''
#url_project: ''
#url_slides: ''
#url_source: ''
#url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: ""
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
# slides: example
---
